{"level":"error","message":"error from wisdom error from wisdom","stack":"Error: error from wisdom\n    at /home/developer/work/learning/npm-demo/Handling&LoggingErrors/routes/users.js:71:9\n    at /home/developer/work/learning/npm-demo/Handling&LoggingErrors/middleware/async.js:4:13\n    at Layer.handle [as handle_request] (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/layer.js:95:5)\n    at next (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/route.js:144:13)\n    at auth (/home/developer/work/learning/npm-demo/Handling&LoggingErrors/middleware/auth.js:14:5)\n    at Layer.handle [as handle_request] (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/layer.js:95:5)\n    at next (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/route.js:144:13)\n    at Route.dispatch (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/route.js:114:3)\n    at Layer.handle [as handle_request] (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/layer.js:95:5)\n    at /home/developer/work/learning/npm-demo/node_modules/express/lib/router/index.js:284:15"}
{"level":"error","message":"error from wisdom error from wisdom","stack":"Error: error from wisdom\n    at /home/developer/work/learning/npm-demo/Handling&LoggingErrors/routes/users.js:71:9\n    at /home/developer/work/learning/npm-demo/Handling&LoggingErrors/middleware/async.js:4:13\n    at Layer.handle [as handle_request] (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/layer.js:95:5)\n    at next (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/route.js:144:13)\n    at auth (/home/developer/work/learning/npm-demo/Handling&LoggingErrors/middleware/auth.js:14:5)\n    at Layer.handle [as handle_request] (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/layer.js:95:5)\n    at next (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/route.js:144:13)\n    at Route.dispatch (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/route.js:114:3)\n    at Layer.handle [as handle_request] (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/layer.js:95:5)\n    at /home/developer/work/learning/npm-demo/node_modules/express/lib/router/index.js:284:15"}
{"level":"error","message":"error from wisdom error from wisdom","stack":"Error: error from wisdom\n    at /home/developer/work/learning/npm-demo/Handling&LoggingErrors/routes/users.js:71:9\n    at /home/developer/work/learning/npm-demo/Handling&LoggingErrors/middleware/async.js:4:13\n    at Layer.handle [as handle_request] (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/layer.js:95:5)\n    at next (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/route.js:144:13)\n    at auth (/home/developer/work/learning/npm-demo/Handling&LoggingErrors/middleware/auth.js:14:5)\n    at Layer.handle [as handle_request] (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/layer.js:95:5)\n    at next (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/route.js:144:13)\n    at Route.dispatch (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/route.js:114:3)\n    at Layer.handle [as handle_request] (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/layer.js:95:5)\n    at /home/developer/work/learning/npm-demo/node_modules/express/lib/router/index.js:284:15"}
{"level":"error","message":"error from wisdom error from wisdom","stack":"Error: error from wisdom\n    at /home/developer/work/learning/npm-demo/Handling&LoggingErrors/routes/users.js:71:9\n    at /home/developer/work/learning/npm-demo/Handling&LoggingErrors/middleware/async.js:4:13\n    at Layer.handle [as handle_request] (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/layer.js:95:5)\n    at next (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/route.js:144:13)\n    at auth (/home/developer/work/learning/npm-demo/Handling&LoggingErrors/middleware/auth.js:14:5)\n    at Layer.handle [as handle_request] (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/layer.js:95:5)\n    at next (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/route.js:144:13)\n    at Route.dispatch (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/route.js:114:3)\n    at Layer.handle [as handle_request] (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/layer.js:95:5)\n    at /home/developer/work/learning/npm-demo/node_modules/express/lib/router/index.js:284:15"}
{"level":"error","message":"error from wisdom error from wisdom","stack":"Error: error from wisdom\n    at /home/developer/work/learning/npm-demo/Handling&LoggingErrors/routes/users.js:71:9\n    at /home/developer/work/learning/npm-demo/Handling&LoggingErrors/middleware/async.js:4:13\n    at Layer.handle [as handle_request] (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/layer.js:95:5)\n    at next (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/route.js:144:13)\n    at auth (/home/developer/work/learning/npm-demo/Handling&LoggingErrors/middleware/auth.js:14:5)\n    at Layer.handle [as handle_request] (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/layer.js:95:5)\n    at next (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/route.js:144:13)\n    at Route.dispatch (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/route.js:114:3)\n    at Layer.handle [as handle_request] (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/layer.js:95:5)\n    at /home/developer/work/learning/npm-demo/node_modules/express/lib/router/index.js:284:15"}
{"level":"error","message":"error from wisdom error from wisdom","stack":"Error: error from wisdom\n    at /home/developer/work/learning/npm-demo/Handling&LoggingErrors/routes/users.js:71:9\n    at /home/developer/work/learning/npm-demo/Handling&LoggingErrors/middleware/async.js:4:13\n    at Layer.handle [as handle_request] (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/layer.js:95:5)\n    at next (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/route.js:144:13)\n    at auth (/home/developer/work/learning/npm-demo/Handling&LoggingErrors/middleware/auth.js:14:5)\n    at Layer.handle [as handle_request] (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/layer.js:95:5)\n    at next (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/route.js:144:13)\n    at Route.dispatch (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/route.js:114:3)\n    at Layer.handle [as handle_request] (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/layer.js:95:5)\n    at /home/developer/work/learning/npm-demo/node_modules/express/lib/router/index.js:284:15"}
{"level":"error","message":"error from wisdom error from wisdom","stack":"Error: error from wisdom\n    at /home/developer/work/learning/npm-demo/Handling&LoggingErrors/routes/users.js:71:9\n    at /home/developer/work/learning/npm-demo/Handling&LoggingErrors/middleware/async.js:4:13\n    at Layer.handle [as handle_request] (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/layer.js:95:5)\n    at next (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/route.js:144:13)\n    at auth (/home/developer/work/learning/npm-demo/Handling&LoggingErrors/middleware/auth.js:14:5)\n    at Layer.handle [as handle_request] (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/layer.js:95:5)\n    at next (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/route.js:144:13)\n    at Route.dispatch (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/route.js:114:3)\n    at Layer.handle [as handle_request] (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/layer.js:95:5)\n    at /home/developer/work/learning/npm-demo/node_modules/express/lib/router/index.js:284:15"}
{"level":"error","message":"error from wisdom error from wisdom","stack":"Error: error from wisdom\n    at /home/developer/work/learning/npm-demo/Handling&LoggingErrors/routes/users.js:71:9\n    at /home/developer/work/learning/npm-demo/Handling&LoggingErrors/middleware/async.js:4:13\n    at Layer.handle [as handle_request] (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/layer.js:95:5)\n    at next (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/route.js:144:13)\n    at auth (/home/developer/work/learning/npm-demo/Handling&LoggingErrors/middleware/auth.js:14:5)\n    at Layer.handle [as handle_request] (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/layer.js:95:5)\n    at next (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/route.js:144:13)\n    at Route.dispatch (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/route.js:114:3)\n    at Layer.handle [as handle_request] (/home/developer/work/learning/npm-demo/node_modules/express/lib/router/layer.js:95:5)\n    at /home/developer/work/learning/npm-demo/node_modules/express/lib/router/index.js:284:15"}
{"level":"error","message":"error from wisdom"}
